contracts/wrapper/NameWrapper.sol

```diff
@@ -20,6 +21,7 @@ error LabelMismatch(bytes32 labelHash, bytes32 expectedLabelhash);
 error LabelTooShort();
 error LabelTooLong(string label);
 error IncorrectTargetOwner(address owner);
+error CannotUpgrade();
```

Comment:
Could this be something more descriptive? If `upgrade()` reverts, I assume it probably cannot be upgraded. Maybe something like `InvalidUpgradeAddress(address address)` would be better.